components/Layouts.tsx

```diff
@@ -76,19 +89,29 @@ const Ellipse2 = styled.div`
   transform: rotate(-45deg);
 `;
 
+
+const Gradient = styled.div`
```

Comment:
`Gradient` is a bit too generic a name IMO. Call it `FFGradient`

Thank you for using a styled component